package.json

```diff
@@ -41,7 +43,7 @@
     "@ledgerhq/hw-app-xrp": "4.48.0",
     "@ledgerhq/hw-transport": "4.48.0",
     "@ledgerhq/hw-transport-http": "4.48.0",
-    "@ledgerhq/live-common": "4.34.1",
+    "@ledgerhq/live-common": "4.35.0",
```

Comment:
I've already merged a branch with `live-common` v4.36.1, be careful not to overwrite it